### PR TITLE
Feat:RDCC-3613:Create skeleton for rd-commondata-dataload

### PR DESCRIPTION
**Pull request by shivajihmcts**
### JIRA link  ###
https://tools.hmcts.net/jira/browse/RDCC-3613

### Change description ###

Changed made to onboard rd-commondata-dataload

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: k8s/namespaces/rd/rd-commondata-dataload/rd-commondata-api.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
```
Comment (shreedhar-hmcts) [inline k8s/namespaces/rd/rd-commondata-dataload/rd-commondata-api.yaml:3]: is file name correct ? since this is data load api ?

**Review by vijayrajagopalan-hmcts — CHANGES_REQUESTED** [review]
Make master pipeline green before merging this PR.

File: k8s/namespaces/rd/rd-commondata-dataload/rd-commondata-api.yaml (hunk 1 above)
Comment (shivajihmcts) [reply k8s/namespaces/rd/rd-commondata-dataload/rd-commondata-api.yaml:3]: renamed file name to rd-commondata-dataload

**Review by hannah38 — APPROVED** [review]

**Review by AdilOoze — DISMISSED** [review]

File: k8s/namespaces/rd/rd-commondata-dataload/rd-commondata-dataload.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  releaseName: rd-commondata-dataload
+  chart:
+    git: git@github.com:hmcts/hmcts-charts
+    ref: master
+    path: stable/rd-commondata-dataload
+  values:
+    java:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      image: hmctspublic.azurecr.io/rd/commondata-dataload:prod-2f2ee01-20211119183711 #{"$imagepolicy": "flux-system:rd-commondata-dataload"}
```
Comment (ManishParyani) [inline k8s/namespaces/rd/rd-commondata-dataload/rd-commondata-dataload.yaml:15]: Does this image exist?

File: k8s/ithc/cluster-01-overlay/rd/kustomization.yaml [hunk 3]
```diff
@@ -4,6 +4,8 @@ namespace: rd
 bases:
 - ../../../namespaces/rd/rd-judicial-data-load/rd-judicial-data-load.yaml
 - ../../../namespaces/rd/rd-location-ref-data-load/rd-location-ref-data-load.yaml
+- ../../../namespaces/rd/rd-commondata-dataload/rd-commondata-dataload.yaml
 patchesStrategicMerge:
 - ../../../namespaces/rd/rd-judicial-data-load/ithc-01.yaml
-- ../../../namespaces/rd/rd-location-ref-data-load/ithc-00.yaml
\ No newline at end of file
+- ../../../namespaces/rd/rd-location-ref-data-load/ithc-00.yaml
```
Comment (SujeethKumarAri) [inline k8s/ithc/cluster-01-overlay/rd/kustomization.yaml:10]: I know this is not as part of your task but can you change this to ithc-01.yaml?

File: k8s/namespaces/rd/rd-commondata-dataload/rd-commondata-dataload.yaml [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+    hmcts.github.com/global-defaults: enabled
+    hmcts.github.com/prod-automated: enabled
+spec:
+  releaseName: rd-commondata-dataload
+  chart:
+    git: git@github.com:hmcts/hmcts-charts
+    ref: master
+    path: stable/rd-commondata-dataload
+  values:
+    java:
```
Comment (SujeethKumarAri) [inline k8s/namespaces/rd/rd-commondata-dataload/rd-commondata-dataload.yaml:15]: This should be `job` I believe. Please refer JRD/LRD data load

**Review by SujeethKumarAri — CHANGES_REQUESTED** [review]

**Review by vijayrajagopalan-hmcts — DISMISSED** [review]

**Review by SujeethKumarAri — DISMISSED** [review]

**Review by vijayrajagopalan-hmcts — DISMISSED** [review]

**Review by ManishParyani — DISMISSED** [review]

File: k8s/namespaces/rd/rd-commondata-dataload/prod-00.yaml [hunk 5]
```diff
@@ -0,0 +1,10 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
```
Comment (timja) [inline k8s/namespaces/rd/rd-commondata-dataload/prod-00.yaml:2]: why are you deploying to prod at the same time as the other environments?

Comment (vijayrajagopalan-hmcts) [reply k8s/namespaces/rd/rd-commondata-dataload/prod-00.yaml:2]: @timja this app is not approved for production deployment yet. We created this file just for setup to make it easier during go live. Happy to remove the prod-* files, but we thought it is harmless.

**Review by vijayrajagopalan-hmcts — APPROVED** [review]

**Review by timja — APPROVED** [review]

**Pull request merged**